deny.toml

```diff
@@ -1,5 +1,5 @@
 [graph]
-all-features = true
```

Comment:
`cargo deny check` can't be run with `all-features` because there is conflicts in the dependencies for `polkavm-contracts` and `wasm-contracts`.